static/src/javascripts/projects/commercial/modules/header-bidding/prebid/prebid.spec.ts

```diff
@@ -19,8 +19,14 @@ jest.mock('../../dfp/get-advert-by-id', () => ({
 }));
 
 jest.mock('../../../../common/modules/experiments/ab', () => ({
+	/**
+	 * Mock defaults to being in control for all tests
+	 * @example
+	 * isInVariantSynchronous(testId, 'control'); // => true
+	 * isInVariantSynchronous(testId, 'variant'); // => false
+	 */
 	isInVariantSynchronous: jest.fn(
-		(_testId, variantId) => variantId === 'variant',
+		(_testId, variantId) => variantId !== 'variant',
```

Comment:
I find this a little counter-intuitive to read. Could we use `mockImplementation()` or  `mockImplementationOnce()` instead to explicitly return true or false in the relevant tests?

Hopefully that won't mean restructuring the test too much otherwise fine to leave as.

Reply:
Good idea, will do!

Reply:
Actually, I think we can get away without mocking the implementation at all

```ts
jest.mock('../../../../common/modules/experiments/ab', () => ({
	isInVariantSynchronous: jest.fn(),
}));
```

Reply:
If that's all we're doing then _I think_ you can remove the stubbed module as Jest should mock the entire module by default.

Reply:
I get an error `define is not defined` when I remove the mock entirely... I assume it's something to do with the implementation imported by `ab-core`?